vendor/inline-scripts/featured-video.js

```diff
@@ -29,7 +29,13 @@
   }
 
   window.canPlayVideo = function () {
```

Comment:
is there a reason why we are not using ember service for such functionality?

Reply:
It's because we want to use it outside ember